requre/helpers/function_output.py

```diff
@@ -40,9 +41,11 @@ def recorded_function(*args, **kwargs):
             # + [x for x in args if isinstance(int, str)] + [f"{k}={v}" for k, v in kwargs.items()]
 
             if STORAGE.is_write_mode:
+                time_before = original_time()
                 output = func(*args, **kwargs)
+                time_after = original_time()
                 STORAGE.store(keys, output)
-
+                DataMiner().metadata = {DataMiner.LATENCY_KEY: time_after - time_before}
```

Comment:
So we are overwriting the global `metadata` in each `store`?

What about adding support for metadata saving to `STORAGE.store`? (And the metadata in the global can be used to overwrite the real latency -- e.g. absolute values or speed  percentage)

Reply:
@lachmanfrantisek no, it rewrites ``metadata`` of the last store item,
in Storage there are another ``metadata`` related to storage file version and similar.
so ``STORAGE`` have also ``metadata`` but currently called   ``requre_internal_object`` :-) could be renamed to ``metadata`` as well

Reply:
you were right, it was possible and it was not hard to do it in this way. My comment were point that STORAGE has also metadata